h2o-algos/src/test/java/hex/grid/GridTest.java

```diff
@@ -59,12 +61,14 @@ public void testParallelModelTimeConstraint() {
               hyperParms,
               new GridSearch.SimpleParametersBuilderFactory(),
               new HyperSpaceSearchCriteria.RandomDiscreteValueSearchCriteria(), 2);
+
       Scope.track_generic(gridSearch);
       final Grid grid = gridSearch.get();
       Scope.track_generic(grid);
 
+      // According to javadoc of `RandomDiscreteValueSearchCriteria.max_runtime_secs`  each model will have one second to finish ( due to params._max_runtime_secs = 1D;) and there is no time limit for Grid
+      // It means if it is enough to finish one model with one second then we should expect whole space to be trained => assertion is probably incorrect
```

Comment:
`params._max_runtime_secs = 1D;`
please use `1d`,  it took me 5 looks to read `1D` instead of `10`.

Reply:
@seb-h2o it is not my code.

Reply:
are you suggesting that the `assertNotEquals` below should be `assertEquals` and that's the reason you want to `@Ignore` the test?
If so, I agree with you, looks like it should be `assertEquals` and the test should be reenabled.

Reply:
@Pscheidl ?

Reply:
@seb-h2o to be consistent with my current understanding of how max_runtime should be managed....yes, it should be `assertEquals`. But probably original intention of the author was different - I assume author wanted to show that 1 sec would not be enough to build all models

Reply:
> @seb-h2o it is not my code.
I know, but as you have the occasion, will change it... it's very minor but a bit disturbing

Reply:
Please fix the test instead of ignoring it, the fix should be part of the PR.

Reply:
@seb-h2o sure, i just wanted to discuss it and then fix

Reply:
The intention was to test whether max runtime will be respected, the training will be stopped and the whole hyperparam space not walked.

Could also be tested by <`grid.modelCount`, instead of notEquals.

Reply:
The second issue is something I can't help with. On Linux, my IntelliJ IDEA uses DejaVu Sans Mono, which can properly distinguish among these characters:

![Screenshot from 2019-12-08 21-40-47](https://user-images.githubusercontent.com/8769110/70395869-97123b80-1a03-11ea-85ad-78455e3ca531.png)

If these basic fonts are not available to you on your Apple device, this one might help you with character readability: https://mplus-fonts.osdn.jp/about-en.html

It's an excellent choice.

Reply:
just for @Pscheidl 
without the comparison above, I quickly read 10, not 1D, I guess it's because I'm used, like most people, to see a digit after another digit if they're not separated by a space...), so font it not the only concern.
Also, you know as much as I do that  in Java the literal suffix is optional for doubles and ints, so no need to specify it if it can lead to confusion.
Besides, again to improve readability, for those suffixes, it's recommended to use the case that leads to less confusion:
- 111L, instead of 111l
- 555f and 555F both work but I think 1st version is more common
- 100d instead of 100D
when you read numbers you haven't written yourself, you just spend a fraction of a second reading those, so I think it's important to make them as readable as possible.